components/mpas-ocean/src/shared/mpas_ocn_diagnostics.F

```diff
@@ -2262,7 +2281,7 @@ subroutine ocn_diagnostic_solve_vortVel( &
 
       ! Compute tangential velocity
 
-      nEdges = nEdgesHalo(2)
```

Comment:
@mark-petersen this is the halo fix for the domain decomp error.  I haven't checked if `nEdgesHalo(3)` would work instead of `nEdgesAll`